apps/graphql/src/dataloaders/itinerariesHelpers.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import type {
+  ApiRouteItemType,
+  ProviderTypeType,
+} from './ItinerariesloaderTypes';
+
+export const getVehicle = (type: ?ProviderTypeType, uniqueNo: string) => ({
+  type: type ?? null,
+  uniqueNo: uniqueNo ?? null,
+});
+
+export const getProvider = (provider: ?string) => ({
+  id: 'ads',
```

Comment:
I didn't find anything like that. 

Btw one more thing – `provider` is probably used on Tequila as a data provider a not as a travel provider, because I found there following row: 
`"check_price_provider": "AMADEUS_MRP",`

Reply:
it's `transporter` now